plugin/src/test/java/io/spine/tools/gradle/bootstrap/ExtensionTest.java

```diff
@@ -58,22 +64,28 @@
 @DisplayName("`spine` extension should")
 class ExtensionTest {
 
+    private static final
+    Correspondence<@NonNull Task, @NonNull TaskName> names = Correspondence.from(
+            (@NonNull Task task, @NonNull TaskName name) -> task.getName().equals(name.name()),
+            "names"
+    );
+
     private PluginTarget pluginTarget;
     private Extension extension;
     private MemoizingSourceSuperset codeLayout;
     private MemoizingDependant dependencyTarget;
     private Path projectDir;
+    private Project project;
 
     @BeforeEach
-    void setUp(@TempDir Path projectDir) {
```

Comment:
Please migrate to [JUnit-based approach](https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory).